src/plugins/testbed/server/index.ts

```diff
@@ -42,6 +43,9 @@ class Plugin {
           return `Some exposed data derived from config: ${config.secret}`;
         })
       ),
+      pingElasticsearch$: startContext.elasticsearch.adminClient$.pipe(
```

Comment:
__note:__ in legacy world, can be used like this:
```ts
kbnServer.core.plugins.get('testbed').pingElasticsearch$.pipe(first()).toPromise().then(() => {
  console.log('Successfully pinged ES!');
}, (err) => {
  console.error('Failed to ping ES!', err);
});
```

If we start using this for real purposes we may change it to smth like: `this.core.testbed`